docs/kyma/04-03-cluster-installation.md

```diff
@@ -331,5 +318,61 @@ For MacOS, run:
     ```
 
 
+  </details>
+  <details>
+  <summary>
+  Gardener
+  </summary>
+
+Install Kyma on a [GKE](https://cloud.google.com/kubernetes-engine/) or [AKS](https://azure.microsoft.com/services/kubernetes-service/) cluster deployed through [Gardener](https://gardener.cloud/).
+
+## Prerequisites
+
+  - [Gardener](https://gardener.cloud/) seed cluster
+  - [Google Cloud Platform](https://console.cloud.google.com/) (GCP) project with Kubernetes Engine API enabled or a [Microsoft Azure](https://azure.microsoft.com) account
+  - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.12.0 or higher
+  - [gcloud](https://cloud.google.com/sdk/gcloud/) or [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
+
+## Choose the release to install
+
+1. Go to [this](https://github.com/kyma-project/kyma/releases/) page and choose the release you want to install.
+
+2. Export the release version as an environment variable. Run:
+
+    ```
+    export KYMA_VERSION={KYMA_RELEASE_VERSION}
+    ```
+
+## Provision a GKE or AKS cluster through Gardener
+
+1. Create a Service Account (SA) with the required permissions in your GKE or AKS project. To learn about the SA requirements for each environment, click the question mark buttons in the **Secrets** tab of your Gardener UI.
```

Comment:
Service Account is the term from GCP. Azure needs credentials and service principals. (Don't mention GKE and AKS in this document at all)